htdocs/core/js/timesheet.js

```diff
@@ -91,17 +91,11 @@ function parseTime(timeStr, dt)
         dt = new Date();
     }
  
-    var time = timeStr.match(/(\d+)(?::(\d\d))?\s*(p?)/i);
+    var time = timeStr.match(/(\d+)(?::(\d\d))?/i);
     if (!time) {
         return -1;
     }
     var hours = parseInt(time[1], 10);
-    if (hours == 12 && !time[3]) {
```

Comment:
This seems to break the calculation of time when lang show time with AM and PM

Reply:
I don't understand why this can show time with AM and PM, these totals are durations and not dates
You can't say, i've spent 1:00 PM hour. So when you spent 12 hours on a task you should see on total per day that you've spent 12 hours in that day and not 0

Reply:
@eldy did you see @ATM-Nicolas' last answer ?